src/components/Countries.js

```diff
@@ -0,0 +1,101 @@
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+import { LinkContainer } from 'react-router-bootstrap';
+import { Col, InputGroup, Row } from 'react-bootstrap';
+import { bindActionCreators } from 'redux';
+import { loadCountries } from '../redux/countries/countries';
+import earth from '../img/earth.png';
+
+const Filter = (props) => {
+  const filterValues = (e) => {
+    props.setFilter(e.target.value);
+  };
+  return (
+    <InputGroup className="mb-3">
+      <span className="input-group-prepend" />
+      <input
+        type="text"
+        onChange={filterValues}
+        className="form-control no-shadow"
+        placeholder="Search Specific Country"
+      />
+    </InputGroup>
+  );
+};
+
+Filter.propTypes = {
+  setFilter: PropTypes.func.isRequired,
+};
+
+const Countries = () => {
+  const dispatch = useDispatch();
+  const loadCountriesAction = bindActionCreators(loadCountries, dispatch);
+  const countries = useSelector((state) => state.countries);
+
+  useEffect(() => {
+    if (countries.length === 0) loadCountriesAction();
+    return () => null;
+  }, []);
+
+  const [filter, setFilter] = useState('');
+  const countryComponents = countries
+    .filter(({ name }) => name.toLowerCase().startsWith(filter.toLowerCase()))
+    .map((country, index) => (
+      <LinkContainer key={country.id} to={`/country/${country.id}`}>
+        <Col
+          xs={12}
+          sm={6}
+          md={6}
+          className={`
+          d-flex flex-column justify-content-between
+          align-items-end
+          ${[1, 0, 0, 1][index % 2] ? 'bg-blue-dark' : 'bg-blue-light'}
+        `}
```

Comment:
- [ ] As requested by the previous reviewer, On mobile version the design says the countries listing should be done in two columns instead of one as per the given design. Kindly update your countries styling to match the given design.


### Your design 
![Screenshot from 2021-10-16 01-03-43](https://user-images.githubusercontent.com/22434660/137544472-ad6b4aa0-db90-456a-9231-da5c65ffb774.png)



### Given design

![Screenshot from 2021-10-16 01-03-56](https://user-images.githubusercontent.com/22434660/137544422-3647fff9-1cc4-4982-839a-48dbf0935b6c.png)

Reply:
Fix column size